cmd/tarocli/assets.go

```diff
@@ -153,6 +154,27 @@ func listUtxos(ctx *cli.Context) error {
 	return nil
 }
 
+var listGroupsCommand = cli.Command{
+	Name:        "groups",
+	ShortName:   "g",
+	Usage:       "list all asset groups",
+	Description: "list all asset groups known to the daemon",
+	Action:      listGroups,
+}
```

Comment:
Maybe the command should be `listgroups` or something like that. Do we expect future other asset group commands?

Reply:
My understanding is that this is a subcommand within `assets` , so the full command would be `assets groups`. Intended to be similar to the `utxos` subcommand.

Reply:
Ah that makes sense!